backend/model/ladybird_export.rb

```diff
@@ -0,0 +1,693 @@
+      # Title {fdid=70}
+      {:header => "{fdid=70}",           :proc => Proc.new {|row| title(row)}},
+      # Date, created {fdid=79}
+      {:header => "{fdid=79}",           :proc => Proc.new {|row| creation_date(row)}},
+      # Physical description {fdid=82}
+      {:header => "{fdid=82}",           :proc => Proc.new {|row| physical_description(row)}},
+      # Language {fdid=84}
+      {:header => "{fdid=84}",           :proc => Proc.new {|row| language(row)}},
+      # Note {fdid=86}
+      {:header => "{fdid=86}",           :proc => Proc.new {|row| note(row)}},
+      # Abstract {fdid=87}
+      {:header => "{fdid=87}",           :proc => Proc.new {|row| abstract(row)}},
+      # Subject, name {fdid=88}
+      {:header => "{fdid=88}",           :proc => Proc.new {|row| name_subjects(row)}},
+      # Subject, topic {fdid=90}
+      {:header => "{fdid=90}",           :proc => Proc.new {|row| topic_subjects(row)}},
+      # Subject, geographic {fdid=91}
+      {:header => "{fdid=91}",           :proc => Proc.new {|row| geo_subjects(row)}},
+      # Genre {fdid=98}
+      {:header => "{fdid=98}",           :proc => Proc.new {|row| nil }}, #BLANK!
+      # Type of resource {fdid=99}
+      {:header => "{fdid=99}",           :proc => Proc.new {|row| nil }}, #BLANK!
+      # Location, YUL {fdid=100}
+      {:header => "{fdid=100}",           :proc => Proc.new {|row| 'Beinecke Rare Book and Manuscript Library, Yale University {id=159091}'}},
+      # Access condition {fdid=102}
+      {:header => "{fdid=102}",           :proc => Proc.new {|row| nil}}, #BLANK!
+      # Restriction {fdid=103}
+      {:header => "{fdid=103}",           :proc => Proc.new {|row| nil }}, #BLANK!
+      # Barcode {fdid=105}
+      {:header => "{fdid=105}",           :proc => Proc.new {|row| barcode(row)}},
+      # YFAD {fdid=106}
+      {:header => "{fdid=106}",           :proc => Proc.new {|row| ead_location(row)}},
+      # Citation {fdid=156}
+      {:header => "{fdid=156}",           :proc => Proc.new {|row| citation_note(row)}},
+      # Item Permission  {fdid=180}
+      {:header => "{fdid=180}",           :proc => Proc.new {|row| nil }}, #BLANK!
+      # Studio Notes {fdid=187}
+      {:header => "{fdid=187}",           :proc => Proc.new {|row| nil }}, #BLANK!
+      # Digital Collection {fdid=275}
+      {:header => "{fdid=275}",           :proc => Proc.new {|row| nil}}, #BLANK!
+      # ISO Date {fdid=280)
+      {:header => "{fdid=280}",           :proc => Proc.new {|row| all_years(row)}},
+      # Content type {fdid=288}
+      {:header => "{fdid=288}",           :proc => Proc.new {|row| nil}}, #BLANK!
+    ]
+  end
+
+  def to_stream
+    p = Axlsx::Package.new
+    wb = p.workbook
+
+    wb.add_worksheet(:name => 'Digitization Work Order') do |sheet|
+      highlight = sheet.styles.add_style :bg_color => "E8F4FF"
+
+      sheet.add_row column_definitions.collect{|col| col.fetch(:header)}
+      dataset.each do |row|
```

Comment:
Comment to mention the side-effect of setting up global data structures on this call